Librarian/SendStatus.js

```diff
@@ -8,96 +8,96 @@ const Book = require('common/Book')()
-							{ "modifiedLog.AddASINAt": { "$lte": moment().subtract(PERIODICAL_DAY, 'days') } }
-						]
-					}
-				]
-			},
-			UpdateUrl: {
-				$and: [
-				  { isKindlized: true },
-				  { isKindlizedUrl: { $ne: true } },
-					{
-						$or: [
-							{ "modifiedLog.UpdateUrlAt": { $exists: false } },
-							{ "modifiedLog.UpdateUrlAt": { "$lte": moment().subtract(PERIODICAL_DAY, 'days') } }
-						]
-					}
-				]
-			}
-		}
-	}
+  constructor(){
+    this.conditions = {
+      RepairImg: {
+        $and: [
+          {
+            $or: [
+              { images: null },
+              { images: { $exists: false } },
+              { $where: "this.images == 0" }
+            ]
+          },
+          {
+            $or: [
+              { "modifiedLog.RepairImgAt": { $exists: false } },
+              { "modifiedLog.RepairImgAt": { "$lte": moment().subtract(PERIODICAL_DAY, 'days') } }
+            ]
+          }
+        ]
+      },
+      InspectKindlize: {
+        $and: [
+          { AuthorityASIN: { $exists: true } },
+          { AuthorityASIN: { $ne: [''] } },
+          { AuthorityASIN: { $ne: null } },
+          { isKindlized: false },
+          {
+            $or: [
+              { "modifiedLog.InspectKindlizeAt": { $exists: false } },
+              { "modifiedLog.InspectKindlizeAt": { "$lte": moment().subtract(PERIODICAL_DAY, 'days') } }
+            ]
+          }
+        ]
+      },
+      AddASIN: {
+        $and: [
+          {
+            $or: [
+              { AuthorityASIN: { $exists: false } },
+              { AuthorityASIN: [''] },
+              { AuthorityASIN: undefined },
+              { AuthorityASIN: 'UNDEFINED' },
+              { AuthorityASIN: ['UNDEFINED'] }
+            ]
+          },
+          {
+            $or: [
+              { "modifiedLog.AddASINAt": { $exists: false } },
+              { "modifiedLog.AddASINAt": { "$lte": moment().subtract(PERIODICAL_DAY, 'days') } }
+            ]
+          }
+        ]
+      },
+      UpdateUrl: {
+        $and: [
+          { isKindlized: true },
+          { isKindlizedUrl: { $ne: true } },
+          {
+            $or: [
+              { "modifiedLog.UpdateUrlAt": { $exists: false } },
+              { "modifiedLog.UpdateUrlAt": { "$lte": moment().subtract(PERIODICAL_DAY, 'days') } }
+            ]
+          }
+        ]
+      }
+    }
+  }
 
-	sentAllStatus(){
-		let val
-		for (val in this.conditions) {
-			if (this.conditions.hasOwnProperty(val)) {
-				let _val = val
-				Book.count(this.conditions[val], function(err, count){
-					if(err){
-						return log.info(err)
-					}
-					Utils.postSlack('[' + _val + ']の調査対象 ' + count + '冊')
-				})
-			}
-		}
-		Book.count(function(err, count){
-			if(err){
-				return log.info(err)
-			}
-			Utils.postSlack('[全書籍]の調査対象 ' + count + '冊')
-		})
-	}
+  sentAllStatus(){
+    let val
+    for (val in this.conditions) {
+      if (this.conditions.hasOwnProperty(val)) {
+        let _val = val
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

*  Missing semicolon.